SplunkRumWorkspace/SplunkRum/SplunkRum/SplunkRum.swift

```diff
@@ -181,20 +181,19 @@ var splunkRumInitializeCalledTime = Date()
 
 /**
  Main class for initializing the SplunkRum agent.
+ 
+   Initialization function.  Call as early as possible in your application, but only on the main thread.
+     - Parameter beaconUrl: Destination for the captured data.
+
+     - Parameter rumAuth: Publicly-visible `rumAuth` value.  Please do not paste any other access token or auth value into here, as this will be visible to every user of your app
+     - Parameter options: Non-required configuration toggles for various features.  See SplunkRumOptions struct for details.
  */
 @objc public class SplunkRum: NSObject {
     static var initialized = false
     static var initializing = false
     static var configuredOptions: SplunkRumOptions?
     static var theBeaconUrl: String?
-   /**
-            Initialization function.  Call as early as possible in your application, but only on the main thread.
-                - Parameter beaconUrl: Destination for the captured data.
-     
-                - Parameter rumAuth: Publicly-visible `rumAuth` value.  Please do not paste any other access token or auth value into here, as this will be visible to every user of your app
-                - Parameter options: Non-required configuration toggles for various features.  See SplunkRumOptions struct for details.
-     
-     */
+
```

Comment:
The change isn't really relevant to this PR, could you remove this part? So only the code change remains

Reply:
done